phase3/all.jsonnet

```diff
@@ -8,4 +9,6 @@ function(cfg)
                if_enabled("heapster", (import "heapster/heapster.jsonnet")(cfg)),
                if_enabled("kube_dns", (import "kube-dns/kube-dns.jsonnet")(cfg)),
                if_enabled("weave_net", (import "weave-net/weave-net.jsonnet")(cfg)),
+               if_cni_plugin("weave", (import "weave-net/weave-net.jsonnet")(cfg)),
```

Comment:
Yeah, I don't know if it's ever come up how many previous versions of Kubernetes that the HEAD of kubernetes-anywhere should support, or what the right pattern is for switching on versions. Creating separate versioned folders for the same addon sounds fine to me, although I don't know if jsonnet would allow you to easily do semver comparisons to automatically choose the correct folder based on the Kubernetes version.

Another option is to always aim to have master/HEAD support the newest stable version of Kubernetes, and we can tag older commits (or use separate branches) for previous versions in case users have reasons to want to use older Kubernetes releases. I'm not a project maintainer, so I don't know what the most palatable solution here is.